src/compas/geometry/brep/brep.py

```diff
@@ -489,6 +487,26 @@ def from_pipe(cls, curve, radius, thickness=None):
         """
         return from_pipe(curve, radius, thickness=thickness)
 
+    @classmethod
+    def from_plane(cls, plane, domain_u=(-1, +1), domain_v=(-1, +1)):
+        """Construct a Brep from a set of planes.
```

Comment:
From one plane of from a set of planes? If the latter, the method should be called `from_planes`